queries.sql

```diff
@@ -14,4 +14,44 @@ select * from animals where neutered = '1';
 select * from animals where not name='Gabumon';
 /* Find all animals with a weight between 10.4kg and 17.3kg 
 (including the animals with the weights that equals precisely 10.4kg or 17.3kg)*/
-select * from animals where weight_kg between 10.4 and 17.3;
\ No newline at end of file
+select * from animals where weight_kg between 10.4 and 17.3;
+
+/* Transaction 1 : update the animals table by setting the species column to unspecified.*/
+begin;
+Update animals set species = 'unspecified';
+rollback;
+
+/* Transaction 2 */
+begin;
+Update animals set species = 'digimon' where name like '%mon';
+Update animals set species = 'pokemon' where species is null;
+commit;
+
+/* Transaction 3 */
+begin;
+delete from animals;
+rollback;
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=After%20the%20rollback%20verify%20if%20all%20records%20in%20the%20animals%20table%20still%20exists), Please verify if all records in the animals table still exist.